commands/src/main/java/org/wildfly/extras/creaper/commands/orb/ChangeOrb.java

```diff
@@ -0,0 +1,606 @@
+                    confidentiality.havingValue() ? Attribute.of(confidentiality.get().value) : confidentiality)
+            .parameter("trustInTarget",
+                    trustInTarget.havingValue() ? Attribute.of(trustInTarget.get().value) : trustInTarget)
+            .parameter("trustInClient",
+                    trustInClient.havingValue() ? Attribute.of(trustInClient.get().value) : trustInClient)
+            .parameter("detectReplay",
+                    detectReplay.havingValue() ? Attribute.of(detectReplay.get().value) : detectReplay)
+            .parameter("detectMisordering",
+                    detectMisordering.havingValue() ? Attribute.of(detectMisordering.get().value) : detectMisordering)
+            .parameter("callerPropagation",
+                    callerPropagation.havingValue() ? Attribute.of(callerPropagation.get().value) : callerPropagation);
+
+        if (orbType == OrbType.WFLY_IIOP) {
+            transform
+                .parameter("supportSsl",
+                        supportSsl.havingValue() ? Attribute.of(supportSsl.get().openjdk) : supportSsl)
+                .parameter("transactions",
+                        transactions.havingValue() ? Attribute.of(transactions.get().openjdk) : transactions)
+                .parameter("security", security.havingValue() ? Attribute.of(security.get().openjdk) : security)
+                .parameter("exportCorbaloc",
+                        exportCorbaloc.havingValue() ? Attribute.of(exportCorbaloc.get().openjdk) : exportCorbaloc)
+                .parameter("addComponentViaInterceptor", addComponentViaInterceptor.havingValue()
+                        ? Attribute.of(addComponentViaInterceptor.get().openjdk) : addComponentViaInterceptor)
+                .parameter("persistentServerId", persistentServerId);
+        } else {
+            transform
+                .parameter("supportSsl",
+                        supportSsl.havingValue() ? Attribute.of(supportSsl.get().jacorb) : supportSsl)
+                .parameter("transactions",
+                        transactions.havingValue() ? Attribute.of(transactions.get().jacorb) : transactions)
+                .parameter("security", security.havingValue() ? Attribute.of(security.get().jacorb) : security)
+                .parameter("exportCorbaloc",
+                        exportCorbaloc.havingValue() ? Attribute.of(exportCorbaloc.get().jacorb) : exportCorbaloc)
+                .parameter("addComponentViaInterceptor", addComponentViaInterceptor.havingValue()
+                        ? Attribute.of(addComponentViaInterceptor.get().jacorb) : addComponentViaInterceptor)
+                .parameter("persistentServerId",
+                        orbType == OrbType.EAP_JACORB ? Attribute.noValue() : persistentServerId);
+        }
+
+        ctx.client.apply(transform.build());
+    }
+
+    static enum OrbType {
+        // jacorb in EAP 6.4.x series and lower
+        EAP_JACORB,
+        // jacorb in WFLY 8 and 9
```

Comment:
So what is it? In `ChangeOrb` javadoc, you say that JacORB is <= WildFly 8 and OpenJDK ORB is WildFly >= 9.